### ISSUE TITLE
Define a custom accuracy metric

### ISSUE THREAD

**Issue by Mithosk93** [keras-team/keras#11540]
Hi everyone!
I'm working with a loss function that excludes all pixels to zero:
```
def ignore_unknown_xentropy(ytrue, ypred):
    return (1-ytrue[:, :, :, 0]) * K.categorical_crossentropy(ytrue, ypred)
```
I noticed that during the training, however, the accuracy continues to consider the pixels to zero. I was wondering if there is a way to define an accuracy dedicated only for some classes?

Thank you guys!
